File: python/f11.parent/Student.py
#!/usr/bin/python
from Person import Person    # include Person.py

# class declaration
class Student(Person):
  ###### Initializer ######
  def __init__(self, name, gpa=0.0):
    super(Student, self).__init__(name)
    self.setGPA(gpa)    # call mutator

  ###### Mutators ######
  def setGPA (self,gpa):
    self.gpa = gpa      # set data

  ###### Accessors ######
  def getGPA(self):
    return self.gpa      # return data

File: ruby/g11.super/Student.rb
#!/usr/bin/ruby
# class declaration
class Student < Person
  ###### Initializer ######
  def initialize(name, gpa=0.0) # constructor
    super(name)        # super's initializer
    setGPA(gpa)        # call mutator
  end

  ###### Mutators ######
  def setGPA (gpa=0.0)
    @gpa = gpa         # set instance data
  end
  
  ###### Accessors ######
  def getGPA
    @gpa               # return instance data
  end
end